web/src/pages/api/v1/oidc/authorize.ts

```diff
@@ -184,12 +185,18 @@ export default async function handleOIDCAuthorize(
   // ANCHOR: Proof is valid, issue relevant codes
   const response = {} as { code?: string; id_token?: string; token?: string };
 
+  // REVIEW: Do we need store code in hybrid flow too? (it can include "code")
```

Comment:
@paolodamico We already passing nonce [here](https://github.com/worldcoin/developer-portal/blob/igorosipov-wid-531-nonce-must-be-received-in-authorization-code-flow-and/web/src/pages/api/v1/oidc/authorize.ts#L216), but only in case one of response_types [equals](https://github.com/worldcoin/developer-portal/blob/igorosipov-wid-531-nonce-must-be-received-in-authorization-code-flow-and/web/src/pages/api/v1/oidc/authorize.ts#L207) `jwt`. 

1. What is `jwt` response_type? I can't find it in the spec.
2. Do we need to change it to `id_token`? 

Also [this](https://github.com/worldcoin/developer-portal/blob/main/web/src/backend/oidc.ts#L13) mapping seems wrong. Shouldn't it be `id_token` & `token` instead of `jwt`?